app/forms/idv/inherited_proofing/va/form.rb

```diff
@@ -2,18 +2,96 @@ module Idv
-                                    address_country]
+            # NOTE: service_provider_error needs to be changed to conform to whatever will
+            # be returned from the service provider (SP) API (the VA in this case); for example:
+            # If the SP returns a payload_hash of
+            # { error: { code: '123', message: 'Error blah, blah' } }, service_provider_error
+            # should be replaced with error_code and error_message.
+            @optional_fields ||= %i[phone
+                                    address_street2
+                                    address_city
+                                    address_state
+                                    address_country
+                                    service_error
+                                    service_provider_error]
+          end
+        end
+
+        before_validation :add_service_errors_if
+        validates(
+          *required_fields, presence: true,
+                            unless: proc { service_error? || service_provider_error? }
+        )
+
+        def add_service_errors_if
+          return unless service_error? || service_provider_error?
+
+          if service_error?
+            errors.add(
+              :service_provider,
+              # Use a "safe" error message for the model in case it's displayed
+              # to the user at any point.
+              I18n.t('inherited_proofing.errors.service_provider.communication'),
+              type: :service_error,
+            )
+          elsif service_provider_error?
+            # NOTE: Add Service Provider API errors as returned in the payload_hash,
+            # defined as optional_fields methods above.
+            # errors.add(:inherited_proofing_service_provider,
+            #   service_provider_error,
+            #   :service_provider_error
+            # )
           end
         end
 
-        validates(*required_fields, presence: true)
+        def service_error?
+          service_error.present?
+        end
+
+        def service_provider_error?
+          # NOTE: Determine if there were any Service Provider API errors
+          # and return true/false.
+        end
+
+        # Base class overrides
+
+        def form_response
+          FormResponse.new(
+            success: valid?,
+            errors: errors,
+            extra: extra || {},
+          )
+        end
+
+        # NOTE: Returning this stops the throttling process cold.
+        # Find out under what circumstances we want to return this.
+        def extra
+          # Use service_error if it's available for more information
+          # in contrast to what is displayed (if ever) to the user via
+          # model errors.
+          # error = service_error || service_provider_error
+          # return if error.blank?
+
+          # {
+          #   proofing_results: {
+          #     exception: error
+          #   }
+          # }
+          nil
+        end
```

Comment:
Don't need to do this override if we're _NOT_ sending throttle-stopping errors via extra -> proofing_results -> exception